stan/math/prim/prob/beta_lccdf.hpp

```diff
@@ -29,43 +29,43 @@ namespace math {
  * @tparam T_scale_fail type of failure parameter
  * @param y (Sequence of) scalar(s) between zero and one
  * @param alpha (Sequence of) success parameter(s)
- * @param beta (Sequence of) failure parameter(s)
+ * @param beta_param (Sequence of) failure parameter(s)
  * @return log probability or sum of log of probabilities
  * @throw std::domain_error if alpha or beta is negative
  * @throw std::domain_error if y is not a valid probability
  * @throw std::invalid_argument if container sizes mismatch
  */
 template <typename T_y, typename T_scale_succ, typename T_scale_fail>
 return_type_t<T_y, T_scale_succ, T_scale_fail> beta_lccdf(
-    const T_y& y, const T_scale_succ& alpha, const T_scale_fail& beta) {
+    const T_y& y, const T_scale_succ& alpha, const T_scale_fail& beta_param) {
```

Comment:
`alpha` and `beta` are the same type of argument here, so either they should both get decorated with `_param` or neither should.  I'd prefer not to decorate conventional names like this, but I won't complain if you decide to add `_param` to `alpha`, too.  Shouldn't `y` then also be decorated with `y_outcome` or something like that?